cvat-ui/src/components/cvat-app.tsx

```diff
@@ -316,7 +319,8 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
                                         <Route exact path='/tasks/create' component={CreateTaskPageContainer} />
                                         <Route exact path='/tasks/:id' component={TaskPageContainer} />
                                         <Route exact path='/tasks/:tid/jobs/:jid' component={AnnotationPageContainer} />
-                                        <Route exact path='/models' component={ModelsPageContainer} />
+                                        { isModelPluginActive
```

Comment:
I agree with Boris. It is better to show instructions how to enable the feature in UI.

Reply:
I think it better to do in separate PR.